build.gradle.kts

```diff
@@ -68,6 +54,9 @@ dependencies {
     implementation(kotlin("stdlib"))
     implementation(kotlin("reflect"))
 
+    implementation("com.sun.xml.ws:jaxws-ri:$jaxRiVersion")
```

Comment:
Tror disse to kan fjernes.

Reply:
Har du prøvd å deploye uten disse og?

Reply:
Har ikke forsøkt å deploye uten disse, bare bygging lokalt.

Reply:
Jeg skal teste det nå.

Reply:
Fungerte bra uten, så fjerner nå 👍